### PR TITLE
dev toools: separate shortcut for electron and for web content

### PR DESCRIPTION
**Pull request by 20after4**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,8 @@
   "license": "GPLv3",
   "devDependencies": {
     "bower": "^1.3.1",
+    "devtron": "^1.4.0",
+    "electron-debug": "~1.0.1",
```
Comment (lorenzhs) [inline package.json]: what are these and why do we need them?

Comment (20after4) [inline package.json]: I don't remember adding them so I guess it must have been something I was experimenting with that made it's way into the patch.

**Comment by 20after4** [comment]
@lorenzhs Ok I got rid of those unrelated package.json changes.

**Comment by lorenzhs** [comment]
Thanks!

**Pull request merged**